Assets/Scripts/Player/Abilities/SpearAbility.cs

```diff
@@ -9,13 +9,15 @@
 using System.Threading;
 using RogueApeStudio.Crusader.UI.Cooldown;
 using RogueApeStudio.Crusader.Audio;
+using log4net.Util;
```

Comment:
Why are you including this?

Reply:
not anymore, it was probably a quick action